components/homme/CMakeLists.txt

```diff
@@ -399,6 +399,8 @@ ENDIF ()
 
 IF (${HOMME_USE_KOKKOS})
   INCLUDE(Kokkos)
+  include_directories(${KOKKOS_INCLUDE_DIR})
+  link_directories(${KOKKOS_LIBRARY_DIR})
```

Comment:
As you said in the description, this is for the external kokkos? Since it potentially changes build on all machines, did you try to build a test? thanks

Reply:
I tried testing with external kokkos, but it did not work as the install was not built with options we needed. However, to even test using external kokkos, I think some cmake changes are needed. I don't recall if this was the reason for this change specifically.

Reply:
Not sure I understand your comment: did external kokkos work with these changes? If not, there is no reason to put in these changes, since they are not verified.